internal/cmd/local/command_kafka.go

```diff
@@ -0,0 +1,23 @@
+package local
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/internal/pkg/cmd"
+)
+
+func NewKafkaCommand(prerunner cmd.PreRunner) *cobra.Command {
+	c := NewLocalCommand(
```

Comment:
Let's make sure we get rid of this style and copy the style of our other commands.